Bottom: 1d76ce396bec17b3b67c2579c6cf5b244116df12
Top:    0e8a04413237e6f89f4709e7bae381af7badbed6
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 14:59:08 +0000

Adjust STEP export to 2.3mm board thickness (Hack - need something quick!)


---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index e4ec9c0..82c89fa 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -72,7 +72,7 @@
 #endif
 
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 #define HACK_COPPER_THICKNESS MM_TO_COORD(0.035)
 #define HACK_PLATED_BARREL_THICKNESS MM_TO_COORD(0.08)
 #define HACK_MASK_THICKNESS MM_TO_COORD(0.01)
diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 47499ce..1d725f3 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -45,7 +45,7 @@
 
 #define CRASH fprintf(stderr, "HID error: pcb called unimplemented STEP function %s.\n", __FUNCTION__); abort()
 
-#define HACK_BOARD_THICKNESS MM_TO_COORD(1.6)
+#define HACK_BOARD_THICKNESS MM_TO_COORD(2.3)
 
 
 HID step_hid;
